src/pages/iou/request/MoneyRequestParticipantsSelector.tsx

```diff
@@ -121,6 +121,7 @@ function MoneyRequestParticipantsSelector({participants = CONST.EMPTY_ARRAY, onF
             undefined,
             undefined,
             iouType === CONST.IOU.TYPE.INVOICE,
+            isCategorizeAction,
```

Comment:
NAB: It would be more flexible to pass the action directly, and then within the function we can check if it's a categorize action. In the future if we need to have logic for other action types we won't need to add another field.